files/glossary.md

```diff
@@ -8,6 +8,11 @@ A front end developer works on "front end" code in an application, commonly mean
 - [Blog post: What is the future of front end development?](https://scale3c.com/blog/what-is-the-future-of-front-end-development-in-2020/)
 
 ## HTML
+When creating a website, HTML is a good language used on web pages. HTML elements is a type of code that made up of different sets of characters. They can also be called as tag. They usually defined as a starting tag which also has an ending tag at the end of the code, and the code between at is the main component of creating the website. HTML elements can have attributes and content, which can be headings, paragraphs, links, lists, embedded media, or any other format.
+	HTML elements can placed into two different groups: block level and inline level. The former makes up the document's structure, while the latter dress up the contents of a block. In addition, Block elements occupy 100% of the available width and have a newline character before and after. Inline elements will only take up the space needed.
+-[Basic HTML Elements](https://www.w3schools.com/html/html_elements.asp)
```

Comment:
make sure there's a space after the `-` or it wont make it a bulleted list. I'll correct this when I merge it but wanted to point that out.